orchestra/proc-macro/src/impl_orchestra.rs

```diff
@@ -175,12 +175,12 @@ pub(crate) fn impl_orchestra_struct(info: &OrchestraInfo) -> proc_macro2::TokenS
 			}
 
 			/// Route a particular message to a subsystem that consumes the message.
-			pub async fn route_message(&mut self, message: #message_wrapper, origin: &'static str) -> ::std::result::Result<(), #error_ty > {
+			pub async fn route_message<P: #support_crate ::Priority>(&mut self, message: #message_wrapper, origin: &'static str) -> ::std::result::Result<(), #error_ty > {
```

Comment:
Is this a breaking change? If so, can it easily be avoided?

Reply:
Since there is already `send_message_with_priority` it might be possible to do the same for `route_message`.

Reply:
Yes, it is breaking,  I guess you are asking in the context of backporting, I think it shouldn't be a problem to get it into stable2506, since that wasn't released yet. 

> Since there is already send_message_with_priority it might be possible to do the same for route_message.

Right, but then you have to copy paste send_message2 and have a send_message_with_priority2.

if we want to get it into stable2503, I can try to copy paste so that I don't modify existing functions, but I'm not sure it is worth the mess.

Reply:
Backports are one reason, another that I just want us to avoid needless breakage in general. We are aiming for more stability.

Why copy paste - can't one call the other? Anyhow, not a blocker - if staying backwards compatible really results in a huge mess, it is not worth it, I don't think Orchestra is used that widely.